fastlane/Fastfile.swift

```diff
@@ -142,20 +150,6 @@ class Fastfile: LaneFile {
         )
     }
 
-    func updateProvisionSettingsLane() {
```

Comment:
Since this is removed, we will need to modify the testing lane https://github.com/nimblehq/ios-templates/blob/bb6ab46d0f41a6048600cfc921d788850418b89a/.github/workflows/test_upload_build_to_test_flight.yml to include the new match and provision methods too.

Now we need to run `syncAppStoreCodeSigningLane` on the workflow instead of `updateProvisionSettingsLane` 🙏